gdal/swig/python/scripts/gdal2tiles.py

```diff
@@ -651,8 +651,15 @@ def __init__(self, arguments ):
         # KML generation
         self.kml = self.options.kml
 
-        # Output the results
+        # LC_CTYPE check
+        if 'UTF-8' not in os.environ.get("LC_CTYPE", ""):
```

Comment:
hum. I thought I had found on the web that LC_CTYPE would be there, for other application like postgres used it... anyway, I don't really know. that's a good idea. I need to figure out how to do that properly. Let me think about it